extension/syntaxes/templateSettingsSyntax.json

```diff
@@ -222,9 +222,49 @@
       "type": "boolean",
       "examples": [],
       "default": true
+    },
+    "postConversionTasks": {
+      "$id": "#root/postConversionTasks",
+      "title": "Post Conversion Tasks",
+      "description": "Specifies commands to be run after conversion is completed. Commands are run in order of definition.",
+      "default": [],
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/postConversionTaskItem"
+      }
     }
   },
   "definitions": {
+    "postConversionTaskItem": {
+      "description": "Defines a command to be executed by VSCode",
+      "type": "object",
+      "uniqueItems": true,
+      "required": [
+        "description",
+        "command"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "description": {
+          "description": "Description of the command.",
+          "type": "string",
+          "minLength": 1
+        },
+        "command": {
```

Comment:
I did consider it for a short moment but decided against it would require the user to lookup parameters which felt unlikely at the moment. Implementing it now would mean that we support it, but would it be used...? I'd say it's a thing for a future iteration, but then again it would make this feature more complete.

Reply:
ah fuck it, I'll do it   :)

Reply:
wait with this! 
Let's do this in it's own PR, otherwise we risk to never get to the end of this one... 😜

Reply:
git checkout -b dev/post-template-task-v2